File: Fifo-Match-IOS-main/Fifo-Match-IOS-main/FIFOMatch/Models/Survey/SurveyModel.swift
//
//  SurveyModel.swift
//  FIFOMatch
//
//  Created by Harendra Singh Rathore on 29/03/22.
//

import Foundation

struct SurveyModel: Decodable {
  let status: Int?
  let message: String?
}

// MARK: - SurveyDetailModel
struct SurveyDetailModel: Decodable {
    let msg, message: String?
    let status: Int?
    let data: SurveyDetailData?
}

// MARK: - SurveyDetailData
struct SurveyDetailData: Decodable {
    let id, userID: Int?
    let heightType, height, bodyType, seeking: String?
    let qualitiesAppreciate, personalityTypes, kids, kidsInFuture: String?
    let myHealth, otherHealth, myQualities, interestedFact: String?
    let decision, preferClarity: String?

    enum CodingKeys: String, CodingKey {
        case id
        case userID = "user_id"
        case heightType = "height_type"
        case height
        case bodyType = "body_type"
        case seeking
        case qualitiesAppreciate = "qualities_appreciate"
        case personalityTypes = "personality_types"
        case kids
        case kidsInFuture = "kids_in_future"
        case myHealth = "my_health"
        case otherHealth = "other_health"
        case myQualities = "my_qualities"
        case interestedFact = "interested_fact"
        case decision
        case preferClarity = "prefer_clarity"
    }
}

struct FillSurveyModel {
  let name: String
  let info: [String]
}

File: Fifo-Match-Application-main/Fifo-Match-Application-main/app/src/main/java/com/fifo/match/model/SurveyModel.kt
package com.fifo.match.model


import com.google.gson.annotations.SerializedName

data class SurveyModel(
    @SerializedName("id")
    val id: Int?,
    @SerializedName("user_id")
    val userId: Int?,
    @SerializedName("height_type")
    val heightType: String?,
    @SerializedName("height")
    val height: Int?,
    @SerializedName("body_type")
    val bodyType: String?,
    @SerializedName("seeking")
    val seeking: String?,
    @SerializedName("qualities_appreciate")
    val qualitiesAppreciate: String?,
    @SerializedName("personality_types")
    val personalityTypes: String?,
    @SerializedName("good_looking")
    val goodLooking: Any?,
    @SerializedName("kids")
    val kids: String?,
    @SerializedName("kids_in_future")
    val kidsInFuture: String?,
    @SerializedName("my_health")
    val myHealth: String?,
    @SerializedName("other_health")
    val otherHealth: String?,
    @SerializedName("relevant")
    val relevant: Any?,
    @SerializedName("my_qualities")
    val myQualities: String?,
    @SerializedName("other_qualities")
    val otherQualities: Any?,
    @SerializedName("personality")
    val personality: String?,
    @SerializedName("interested_fact")
    val interestedFact: String?,
    @SerializedName("decision")
    val decision: String?,
    @SerializedName("prefer_clarity")
    val preferClarity: String?,
    @SerializedName("created_at")
    val createdAt: String?,
    @SerializedName("updated_at")
    val updatedAt: String?,
    @SerializedName("deleted_at")
    val deletedAt: Any?
)